### ISSUE TITLE
IPv6 Support

### ISSUE THREAD

**Issue by LucasSymons** [DavidMoore/ipfilter#99]
I was wondering if there is any chance of adding a way to support IPv6 in the block lists with it becoming more and more popular. 

I was looking at iblocklist and it looks like all the IPv6 lists are subscription-only, So I am wondering if there is a way for me to use my subscription to access the lists. That way we don't breach their TOS.

**Comment by DavidMoore** [comment]
The rewritten version should support IPv6, but I'm not sure how much work it would be to update the current code to support IPv6.

Do you have an example of what the IPv6 lists look like? Just a couple of lines would be fine.

I need to document how to update the config to use your custom lists. Or even better just build that into the UI.

**Comment by LucasSymons** [comment]
Yeah, tricky they are all locked behind a subscription and I don't want to sign up if it won't be supported haha. 
Let me ask their support
